### PR TITLE
Added support to change timezone from settings page(#1x69rpc)

### PR DESCRIPTION
**Pull request by Yashi002**

### CODE REVIEW

File: src/views/Settings.vue [hunk 1]
```diff
@@ -123,21 +125,23 @@ export default defineComponent({
...
     },
+    setFacility (facility: any) {
+      if (this.userProfile)
+        this.store.dispatch('user/setFacility', {
+          'facility': this.userProfile.facilities.find((fac: any) => fac.facilityId == facility['detail'].value)
+        });
+    },
     logout () {
       this.store.dispatch('user/logout').then(() => {
-        this.store.dispatch('picklist/clearPicklist')
```
Comment (adityasharma7) [inline src/views/Settings.vue:128]: I think this is not related. We should revert.

File: src/views/Settings.vue [hunk 2]
```diff
@@ -123,21 +125,23 @@ export default defineComponent({
       });
       return popover.present();
     },
+    setFacility (facility: any) {
```
Comment (adityasharma7) [inline src/views/Settings.vue:128]: We should not change the position. Move it after logout and only specific changes should be visible

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

File: src/views/Settings.vue (hunk 1 above)
Comment (disha1202) [reply src/views/Settings.vue]: Reverted this change.

File: src/views/Settings.vue (hunk 2 above)
Comment (disha1202) [reply src/views/Settings.vue]: Made the required changes.

**Review by adityasharma7 — APPROVED** [review]
